Package exports are misconfigured for use in TypeScript projects

When using this package in a TypeScript project the compiler complains with the error message

```
Cannot find module '@shopify/i18next-shopify' or its corresponding type declarations.
```

Even though this package itself doesn't export typescript definitions, I think this may still indicate a misconfiguration with either the build process, or one of the fields in package.json which the TypeScript compiler is looking.

This may be related to the `main: ./index.js` definition in the package.json which is pointing to an invalid file (./index.js does not exist). See https://publint.dev/@shopify/i18next-shopify@0.2.0 for more detail

--- a/.changeset/lucky-shrimps-leave.md
+++ b/.changeset/lucky-shrimps-leave.md
@@ -1,0 +1,5 @@
+---
+'@shopify/i18next-shopify': patch
+---
+
+Fixes main field in package.json for TypeScript and bundlers

--- a/package.json
+++ b/package.json
@@ -2,7 +2,7 @@
   "name": "@shopify/i18next-shopify",
   "version": "0.2.0",
   "description": "i18nFormat plugin to use Shopify's translation format with i18next",
-  "main": "./index.js",
+  "main": "./dist/commonjs/index.js",
   "module": "dist/es/index.js",
   "keywords": [
     "i18next",

